client/components/empty-content/style.scss

```diff
@@ -0,0 +1,10 @@
+/** @format */
+
+.woocommerce-empty-content {
+	text-align: center;
+
+	.woocommerce-empty-content__action.is-primary {
+		color: $white;
+		margin-right: $gap;
```

Comment:
This is adding an offset when there's just a single button, you could do this instead to make sure you're putting space between two buttons:

```
.woocommerce-empty-content__actions {
   .components-button + .components-button {
      margin-left: $gap;
   }
}
```